contracts/cw20-merkle-airdrop/src/msg.rs

```diff
@@ -0,0 +1,170 @@
+use crate::ContractError;
+use cosmwasm_std::{from_slice, Binary, Uint128};
+use cw_utils::{Expiration, Scheduled};
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub struct InstantiateMsg {
+    /// Owner if none set to info.sender.
+    pub owner: Option<String>,
+    pub credits_address: Option<String>,
+    pub reserve_address: Option<String>,
+    pub neutron_denom: String,
```

Comment:
why not a const? we know the denom for sure

Reply:
Hmmm, yep, I think I can hardcode "untrn" into contract.

Reply:
Done!